modules/packages/LinearAlgebra.chpl

```diff
@@ -221,6 +222,263 @@ class LinearAlgebraError : Error {
     }
 }
 
+pragma "no doc"
+/*
+This class promotes lazy computation of required
+Helper matrices to evaluate matrix exponential.
+
+High precision of expm requires smaller computations,
+hence we do a lazy computation depending on the norm value
+of the matrix.
+*/
+class ExpmPadeHelper {
+  type matType;
+  var A : matType;
+  var A2 : matType;
+  var A4 : matType;
+  var A6 : matType;
+  var A8 : matType;
+  var A10 : matType;
+  var Exactd4 : real;
```

Comment:
I think it makes sense for all the `A*` fields to be capitalized, but this field is using PascalCase instead of camelCase.  It should use the capitalization style that `isAComputed` uses (camelCase)